import_flow_events.py

```diff
@@ -118,26 +265,31 @@ def import_events(force_reload=False):
         for day in days_to_load:
             print "LOADING", day
             # Clear any existing data for that day, to avoid duplicates.
-            db.run(Q_CLEAR_EVENTS.format(timestamp=day_to_timestamp(day)))
+            db.run(Q_CLEAR_DAY_METADATA.format(day=day))
+            db.run(Q_CLEAR_DAY_EVENTS.format(day=day))
             s3path = EVENTS_FILE_URL.format(day=day)
-            db.run(Q_COPY_EVENTS.format(
+            db.run(Q_COPY_CSV.format(
                 s3path=s3path,
                 **CONFIG
             ))
+            db.run(Q_INSERT_METADATA)
+            db.run(Q_UPDATE_BEGIN_TIME)
+            db.run(Q_UPDATE_DURATION)
+            db.run(Q_UPDATE_COMPLETED)
+            db.run(Q_UPDATE_NEW_ACCOUNT)
+            db.run(Q_INSERT_EVENTS)
+            db.run(Q_UPDATE_TIMESTAMP)
 
         # Print the timestamps for sanity-checking.
-        print "MIN TIMESTAMP", db.one("SELECT MIN(timestamp) FROM flow_events")
-        print "MAX TIMESTAMP", db.one("SELECT MAX(timestamp) FROM flow_events")
+        print "MIN TIMESTAMP", db.one("SELECT MIN(timestamp) FROM temporary_raw_flow_data")
+        print "MAX TIMESTAMP", db.one("SELECT MAX(timestamp) FROM temporary_raw_flow_data")
+
+        db.run(Q_DROP_CSV_TABLE)
```

Comment:
Should we drop the import table after each iteration of the loop, rather than once at the end?  Otherwise ISTM that the `Q_UPDATE_BEGIN_TIME`, `Q_UPDATE_DURATION` etc could be operating on the same data multiple times.